packages/firebase_in_app_messaging/example/lib/main.dart

```diff
@@ -0,0 +1,122 @@
+import 'dart:async';
+
+import 'package:firebase_analytics/firebase_analytics.dart';
+import 'package:firebase_analytics/observer.dart';
+import 'package:firebase_in_app_messaging/firebase_in_app_messaging.dart';
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  static FirebaseAnalytics analytics = FirebaseAnalytics();
+  static FirebaseAnalyticsObserver observer =
+      FirebaseAnalyticsObserver(analytics: analytics);
```

Comment:
It doesn't seem like you're doing anything with `observer` here. In order for the observer to be useful you have to pass it to the `navigatorObservers` of `MaterialApp`.

Reply:
Nice catch - removed it.